### PR TITLE
Release 0.10.0

### PR DESCRIPTION
**Pull request by e5l**

### CODE REVIEW

**Comment by rsinukov** [comment]
Lgtm

File: src/commonTest/kotlin/EscapeAppendTest.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import kotlinx.html.stream.escapeAppend
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class EscapeAppendTest {
```
Comment (karollewandowski) [inline src/commonTest/kotlin/EscapeAppendTest.kt:5]: Please add edge-case tests for:
- `"\\&&"`  -> `"&&amp;"`
- `"\\&amp;"` -> `"&amp"`
- `"text\\"` -> throws exception

Also, it would be good to test regular non-escaped characters. Like:
- `"\\a"`  -> `"a"`

If this is expected.

And maybe more I missed?
